src/contrib/cluster/Akka.Cluster.Sharding/reference.conf

```diff
@@ -184,8 +184,10 @@ akka.cluster.sharding {
   distributed-data {
     # minCap parameter to MajorityWrite and MajorityRead consistency level.
     majority-min-cap = 5
-
-    durable.keys = ["shard-*"]
```

Comment:
Do you recall why we added this in the first place?

Reply:
It was in the official JVM reference.conf, I didn't know that our default value is different than theirs

Reply:
Well it seems that our issue really comes down to the .NET LightningDb packages not having all of the native dependencies they need. We'll add this setting back when we move the default for Akka.Cluster.Sharding state store mode away from persistence and to DData.